utils/credential_utility/roles/update_config/vars/main.yml

```diff
@@ -63,9 +63,9 @@ omnia_credentials:
       password:
         - docker_password
   slurm:
-    default:
+    mandatory:
```

Comment:
slurm user name also to be taken

Reply:
this is the username for the db  previously it was not part of omnia_config